src/App.jsx

```diff
@@ -14,57 +16,65 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectedGood, setSelectedGood] = useState(goods[8]);
+  const isSelectedGood = good => selectedGood === good;
```

Comment:
let's define this like variable in `.map()` block so we don't need call this as function every time